pyvrp/cpp/ProblemData.cpp

```diff
@@ -233,11 +235,16 @@ ProblemData::VehicleType::VehicleType(size_t numAvailable,
     if (std::any_of(capacity.begin(), capacity.end(), isNegative<Load>))
         throw std::invalid_argument("capacity amounts must be >= 0.");
 
-    if (twEarly > twLate)
-        throw std::invalid_argument("tw_early must be <= tw_late.");
+    // Note that latestStart <= latestFinish is not required, so a route can
+    // start after its latest finish time (but this will result in time warp).
```

Comment:
Should we then not just require it when validating the input data?

Reply:
It depends on preference. The question is whether we should enforce the `latestStart` to be no later than the `latestFinish`. Of course, it can never be feasible to start a route after the `latestFinish/twLate`, but in that situation there will always be time warp anyway for finishing after the `latestFinish/twLate`. So, in that sense it is not needed to enforce that `latestStart <= latestFinish`. Please let me know what you prefer.

Reply:
Sorry, I forgot to return to this. I think we should require `startLate <= twLate`, because:
- It comes from the user; now that the value is default optional. If it's not passed we already do the right thing.
- The user is clearly doing something nonsensical if they provide an explicit `startLate` such that `startLate > twLate`.